tools/cloud-build/daily-tests/ansible_playbooks/test-mounts-and-partitions.yml

```diff
@@ -32,27 +41,17 @@
   ansible.builtin.fail:
```

Comment:
Line above this is incorrect (probably a prior mistake of mine). While your changes these, could you change the name of the task to "Check partitions exist"? If you'd rather, I can create a quick bugfix PR for that instead.

Reply:
Done.